MedvedevKS/lab1/calculator/calculator.cpp

```diff
@@ -0,0 +1,50 @@
+#include "stdafx.h"
+#include "iostream"
+#include "conio.h"
+using namespace std;
+
+int main()
+{
+	float left, right, result; char sign;
```

Comment:
Do not declare variables with different types at the same line of code, because it become harder to read

Reply:
got it